cmd/svcat/output/binding.go

```diff
@@ -97,3 +97,16 @@ func WriteAssociatedBindings(w io.Writer, bindings []v1beta1.ServiceBinding) {
 	}
 	t.Render()
 }
+
+// WriteDeletedBindingNames prints the names of a list of bindings
+func WriteDeletedBindingNames(w io.Writer, bindings []v1beta1.ServiceBinding) {
+	for _, binding := range bindings {
+		WriteDeletedBindingName(w, binding.Name)
+	}
+}
+
+// WriteDeletedBindingName prints the name of a binding
+func WriteDeletedBindingName(w io.Writer, bindingName string) {
+	message := fmt.Sprintf("deleted %s", bindingName)
```

Comment:
tiny nit: This could be simplified to just use `fmt.Fprintf`

Reply:
Will do! I replaced this with: 

```
fmt.Fprintf(w, "deleted %s\n", bindingName)
```